From 54c7fafd406c72c9f56893bc5e82af311b29de80 Mon Sep 17 00:00:00 2001
From: Anthoine Bourgeois <anthoine.bourgeois@vates.tech>
Date: Tue, 18 Mar 2025 19:28:22 +0100
Subject: [PATCH] tapdisk: Replace structure name in sizeof

xenio_blkif_req structure doesn't exist anymore, blkif_request_t is the
right structure here. Hopefully a pointer is a pointer in any case.

No functional change expected.

Signed-off-by: Anthoine Bourgeois <anthoine.bourgeois@vates.tech>
---
 drivers/td-req.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/td-req.c b/drivers/td-req.c
index c35b02c6..7b421d87 100644
--- a/drivers/td-req.c
+++ b/drivers/td-req.c
@@ -943,7 +943,7 @@ tapdisk_xenblkif_reqs_init(struct td_xenblkif *td_blkif)
     }
 
     td_blkif->reqs_free =
-        malloc(td_blkif->ring_size * sizeof(struct xenio_blkif_req *));
+        malloc(td_blkif->ring_size * sizeof(struct blkif_request_t *));
     if (!td_blkif->reqs_free) {
         err = -errno;
         goto fail;
